### ISSUE TITLE
NVDA announces actionable items after navigating away not while action is being performed

### ISSUE THREAD

**Issue by khushveer** [nvaccess/nvda#11231]
### Steps to reproduce:
Open drop down in application
Move to the button that expands the list in the drop down
navigate within the menu
The action "expanded" is announced after navigating away from the button

### Actual behavior:
NVDA announces the action performed on any item in dropdown not while performing that action, but after navigating away from it

### Expected behavior:
NVDA should announce the action performed on any item in dropdown while performing that action, not after navigating away from it

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2020.1

#### Windows version:
10.0.18363 Build 18363

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Adriani90** [comment]
Hello,
Please provide on all your issues also an example website or a minimal test case or code snippet so that we can reproduce the issues. Otherwise it will not be possible to investigate further.
And please also specify the browser in the issue description in which you reproduce the issue.

**Comment by DrSooom** [comment]
@Adriani90 meant issue 11229 and 11230. As I saw this issue here and his comment on it after these two issues, I maybe posted to early a comment in the two other issues. … Can happen. 😉

**Comment by Adriani90** [comment]
We don't have any example to reproduce. I assume there is no aria has popup on the button which expands or colapse the list, or any other kind of aria atribute which indicates the status when pressing enter on the button. Given the missing test case, investigation is not possible. I am closing. For more details, you can also look at #9096.
